modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Executor.scala

```diff
@@ -126,7 +129,7 @@ object Executor { self =>
       (liftA(liftT(go)) |@| liftA(gets(_.nextStep.nonEmpty)))(_ && _)
 
     // Run to completion, or to error, as long as not cancelled or complete
-    def runT(go: Task[Boolean], saveState: ExecState => Task[Unit]): ExecActionF[Unit] =
+    def runT[A](go: Task[Boolean], saveState: ExecState => Task[A]): ExecActionF[Unit] =
```

Comment:
Right, I was trying to obtain the `ExecState` out of this call instead of returning `Task[Unit]` but it may not be necessary now that I took another route